spec/system/candidate_interface/candidate_withdraws_spec.rb

```diff
@@ -26,6 +28,8 @@ def given_i_am_signed_in_as_a_candidate
   def and_i_have_an_application_choice_awaiting_provider_decision
     form = create(:completed_application_form, :with_completed_references, candidate: current_candidate)
     @application_choice = create(:application_choice, :awaiting_provider_decision, application_form: form)
+    @provider_user = create(:provider_user)
```

Comment:
This works too:

```suggestion
    @provider_user = create(:provider_user, providers: [@application_choice])
```

Reply:
Minor tweak to make it work:
`@provider_user = create(:provider_user, providers: [@application_choice.provider])`